dockerfiles/alpine/is/Dockerfile

```diff
@@ -106,14 +108,9 @@ RUN \
     && apk add unzip \
     && apk add wget
 
-## set the user and work directory
-USER ${USER_ID}
-WORKDIR ${USER_HOME}
-
-COPY --chown=wso2carbon:wso2 ${WSO2_SERVER}.zip ${USER_HOME}/
-
 RUN \
-    unzip -d ${USER_HOME} ${WSO2_SERVER}.zip \
```

Comment:
Lets not remove these lines as this section is used to allow copying zip files to docker context

Reply:
Check other places and update